File: ControlRemote/Web/ClientApp/src/app/Dto/UserCreateModel.ts
export class UserCreateModel {
    name: string;
    login: string;
    password: string;
    role: string;

    public constructor(_name: string, _login: string, _password: string, _role: string) {
        this.name = _name;
        this.login = _login;
        this.password = _password;
        this.role = _role;
    }
}

File: ControlRemote/Web/Dto/UserCreateModel.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class UserCreateModel
    {
        public string Name { get; set; }
        public string Login { get; set; }

        [DataType(DataType.Password)]
        public string Password { get; set; }
        public string Role { get; set; }
    }
}
